### PR TITLE
BM:FAQ on Basemaps

### PR DESCRIPTION
**Pull request by bmalina**
Closes issue 194 @msschroeder review and merge.

### CODE REVIEW

**Comment by rowo** [comment]
@bmalina removed the lines i saw. let me know if i missed anything

File: faqs/_posts/2016-10-27-faq-basemaps.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ![](/content/screenshots/faq/faq-basemap-6.png)
+   
+3. **Name** - The name of your basemap. This will be displayed on the list of available maps for your form.
+4. **Attribution** - (Optional) Any attribution to the appropriate organization(s) owning the copyright to the basemap. This attribution will appear in the map view. 
+
+**Save** the information entered.
+
+  ![](/content/screenshots/faq/faq-basemap-7.png)
+  
+Now that your basemap is linked to your form, you can view it by going to the form’s Map View and selecting the layer button in the bottom left as shown below;
```
Comment (msschroeder) [inline faqs/_posts/2016-10-27-faq-basemaps.md]: **Map View** (Bold this)

File: faqs/_posts/2016-10-27-faq-basemaps.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  ![](/content/screenshots/faq/faq-basemap-9.png)
+  
+Select your basemap from the list.
+
+  ![](/content/screenshots/faq/faq-basemap-10.png)
+ 
+
+>**Note:**<br/>
+>
+Ona now supports vector tiles on all basemaps. Vector tiles are able to deliver large amounts of geographic data in small chunks to a browser or client app. This results in a few key benefits:
```
Comment (msschroeder) [inline faqs/_posts/2016-10-27-faq-basemaps.md]: delete now

File: faqs/_posts/2016-10-27-faq-basemaps.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+<br><br>
+   * Styling - as vectors, tiles can be styled when requested allowing one to create maps using many styles on global data;
```
Comment (msschroeder) [inline faqs/_posts/2016-10-27-faq-basemaps.md]: Capitalize A in As

**Pull request merged**